client/client.go

```diff
@@ -152,6 +155,17 @@ type Client struct {
 	warningTimestamp time.Time
 
 	getWebsocket getWebsocketFunc
+
+	pebbleDir  string
+	socketPath string
+}
+
+func (c *Client) PebbleDir() string {
+	return c.pebbleDir
+}
+
+func (c *Client) SocketPath() string {
+	return c.socketPath
 }
```

Comment:
I opted for "private fields and public accessors" here on purpose to make it clear that these two members are not supposed to be changed from the outside after the client has been created.